### DISCUSSION TITLE
add docker-compose file support along with the JSON siles

### DISCUSSION THREAD

**Discussion by m3t41** [containers/fetchit#322] [Ideas]
I like the fact that i can use kubernetes Minifest YAMl Files also. but its nice and handy when i can also use all of the 1000 and 100.000 docker-compose Files out there.

So is there a change to implement docker-compose compatiblity also as a new method?

**Comment by cooktheryan** [comment]
when we were initially coding this the golang option for podman wasn't available for this feature
